service/lib/agama/manager.rb

```diff
@@ -205,18 +205,22 @@ def valid?
 
     # Collects the logs and stores them into an archive
     #
-    # @param user [String] local username who will own archive
+    # @param path [String] directory where to store logs
+    #
     # @return [String] path to created archive
-    def collect_logs(user)
-      output = Yast::Execute.locally!("save_y2logs", stderr: :capture)
-      path = output[/^.* (\/tmp\/y2log-\S*)/, 1]
-      Yast::Execute.locally!("chown", "#{user}:", path)
+    def collect_logs(path: nil)
+      opt = "-d #{path}" if !path.nil? && !path.empty?
 
-      path
+      `agama logs store #{opt}`.strip
     end
 
     # Whatever has to be done at the end of installation
     def finish_installation
+      # TODO: make it optional
```

Comment:
I do not think we are going to make it optional but, for sure, it will not happen just because of this comment :-) If it is important, we should create a Trello card or open an issue.

Reply:
it was just an idea which i didn't want to forget of. I'll remove it if you don't think that can be useful